i6_models/parts/conformer/convolution.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+import torch
+from torch import nn
+from i6_models.config import ModelConfiguration
+from typing import Callable, Union, Any, Type
+
+
+@dataclass
+class ConformerConvolutionV1Config(ModelConfiguration):
+    channels: int
```

Comment:
Please add documentation to the parameters, similar to #6

Also lets talk about the defaults for activation and dropout in the meeting in 15 minutes.

Reply:
why i need to add documentation here. it is already documented in the class constructor. this is only used for `ConformerConvolutionV1` class. so this would be a duplicate.

Reply:
Is it? I think you removed it when changing to the config format.
Also I think it makes sense to have this here so when building the config e.g. PyCharm shows the doc.

Reply:
oh it was removed yes.